pdf-service/LOCALSETUP.md

```diff
@@ -0,0 +1,37 @@
+# Local Setup
+
+To setup the PDF-service in your local system, clone the [Core Service repository](https://github.com/egovernments/core-services).
+
+## Dependencies
+
+### Infra Dependency
+
+- [x] Postgres DB
+- [ ] Redis
+- [ ] Elasticsearch
+- [x] Kafka
+  - [x] Consumer
+  - [x] Producer
+
+## Running Locally
+
+### Local setup
+1. To setup the PDF-service, clone the [Core Service repository](https://github.com/egovernments/core-services)
+2. Open the terminal and run the following command
```

Comment:
which command?

Reply:
Updated the localsetup file, this line was added by mistake